### PR TITLE
Add Hachisuka Scans presence

### PR DESCRIPTION
**Pull request by Jetee0**
Home Page print
image

Manga Page Print
image

Read Page Print
image

### CODE REVIEW

**Comment by Yuujet** [comment]
Lol CodeFactor hates me, it's not possible kekw

File: websites/H/Hachisuka Scans/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "service": "Hachisuka Scans",
+  "description": {
+    "en": "Read manga online at Hachisuka Scans without ads, high quality images.",
+    "pt_BR": "Leia mangá online na Hachisuka Scans sem anúncios, imagens de alta qualidade.",
+    "pt": "Leia mangá online na Hachisuka Scans sem anúncios, imagens de alta qualidade.",
+    "nl": "Lees manga online op Hachisuka Scans zonder advertenties, afbeeldingen van hoge kwaliteit."
+  },
+  "url": "hachisukascans.tk",
+  "version": "1.0.8",
```
Comment (Bas950) [inline websites/H/Hachisuka Scans/dist/metadata.json:14]:
```suggestion
  "version": "1.0.0",
```

File: websites/H/Hachisuka Scans/presence.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+      const pesquisaTexto = document.querySelector("body > div.wrap > div > div > div.c-search-header__wrapper > div > div.search-content > form > input.search-field.manga-search-field.ui-autocomplete-input");
+
+      presenceData.details = "Pesquisando por";
+      presenceData.state = pesquisaTexto.value;
+      presenceData.smallImageKey = "search";
+      presenceData.smallImageText = "Pesquisando";
+    } else {
+      presenceData.details = "Página inicial";
+      presenceData.buttons = [{label: "Hachisuka Scans",url: document.URL}];
```
Comment (Bas950) [inline websites/H/Hachisuka Scans/presence.ts:21]: buttons are not allowed to direct to the homepage

File: websites/H/Hachisuka Scans/presence.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        presenceData.details = `Postando [ ${postMangaType.value} ]`; 
+        presenceData.state = postMangaName.value;
+      } else {
+        presenceData.details = `Postando [ ${postMangaType.value} ] ( ${postMangaCap.value} )`; 
+        presenceData.state = postMangaName.value;
+      }
+      presenceData.buttons = [{label: `Página do ${postMangaName.value.slice(0, 15)}...`,url: postMangaURL.textContent}];
+    } else {
+      presenceData.details = null;
+      presenceData.state = null;
```
Comment (Bas950) [inline websites/H/Hachisuka Scans/presence.ts:47]: never set null, use 
delete presenceData.details

**Review by Bas950 — CHANGES_REQUESTED** [review]
Use classnames in querySelector to get the text you want, right now it's just so unreadable

**Review by Bas950 — CHANGES_REQUESTED** [review]
Please run prettier

**Comment by SlowLife1661** [comment]
Hi, I'm closing this as it has been inactive over 7 days

**Pull request closed without merging**